docs/core/porting/breaking-changes.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Breaking changes
+description: Breaking changes between releases of .NET and how it impacts compatibility.
+author: stebon
+ms.date: 06/14/2021
+---
+# Breaking Changes
+
+Visit the following documentation to learn about and find breaking changes when moving from the .NET Framework to .NET and between releases of .NET.
+
+## [What are breaking changes?](../compatibility/index.md)
+
+## [Types of compatibility.](../compatibility/categories.md)
+
+## [Find breaking changes.](../compatibility/breaking-changes.md)
```

Comment:
This needs more content. We wouldn't want to link to another article inside of a header. Have the header, but there needs to be more information, like a paragraph explaining the section concept, then use something like:

`For more information, see LINK` which is our typical wording for linking people to more info.

Reply:
Will do.

Reply:
Done